lakefile.lean

```diff
@@ -10,23 +10,23 @@ lean_exe ffi where
 def ffiC := "ffi.c"
 def ffiO := "ffi.o"
 
-target importTarget (pkg : Package) : FilePath := do
-  let oFile := pkg.oleanDir / ffiO
+target importTarget (pkg : NPackage _package.name) : FilePath := do
+  let oFile := pkg.buildDir / ffiO
   let srcJob ← inputFile ffiC
   buildFileAfterDep oFile srcJob fun srcFile => do
     let flags := #["-I", (← getLeanIncludeDir).toString]
     compileO ffiC oFile srcFile flags
 
-extern_lib libffi (pkg : Package) := do
+extern_lib libffi (pkg : NPackage _package.name) := do
   let name := nameToStaticLib "ffi"
   let job ← fetch <| pkg.target ``importTarget
-  buildStaticLib (pkg.libDir / name) #[job]
+  buildStaticLib (pkg.buildDir / name) #[job]
```

Comment:
I'm also not 100% confident that `buildDir` is the best directory to be using here